save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -66,13 +64,34 @@ class DownloadFilesController(
         additionalToolsFileSystemRepository.getFileInfoList(ProjectCoordinates(organizationName, projectName))
     }
 
+    /**
+     * @param organizationName
+     * @param projectName
+     * @param dirName
+     * @param authentication
+     * @return [Mono] with response
+     */
+    @DeleteMapping(path = ["/api/$v1/files/{organizationName}/{projectName}/{dirName}/delete"])
+    @Suppress("UnsafeCallOnNullableType")
+    fun delete(
+        @PathVariable organizationName: String,
+        @PathVariable projectName: String,
+        @PathVariable dirName: String,
+        authentication: Authentication,
+    ) = projectService.findWithPermissionByNameAndOrganization(
+        authentication, projectName, organizationName, Permission.WRITE
+    ).map {
+        additionalToolsFileSystemRepository.deleteFileByDirName(ProjectCoordinates(organizationName, projectName), dirName)
+        ResponseEntity.ok("File deleted successfully")
+    }
+
     /**
      * @param fileInfo a FileInfo based on which a file should be located
      * @param organizationName
      * @param projectName
      * @return [Mono] with file contents
      */
-    @GetMapping(path = ["/api/$v1/files/{organizationName}/{projectName}/download"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
+    @PostMapping(path = ["/api/$v1/files/{organizationName}/{projectName}/download"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
```

Comment:
Let's maybe unify paths of download and delete methods? Then it can still be `GET` with the same `createdAt` path variable as `DELETE`

Reply:
Didn't change, I don't use this method anymore